website/docusaurus.config.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = [
```

Comment:
Actually the users page can be useful for companies who want to promote their products. Maybe we should even add Docusaurus as a user here :)

Reply:
nice idea!

Reply:
I'm not sure to understand the usecase. 

Which company would want to promote their product on this page exactly? 

Maybe the project could have a paid sponsorship or something and a company would advertise here?

Anyway, I'll keep this page if you feel it can be useful some day :D